### ISSUE TITLE
Adding columns to query

### ISSUE THREAD

**Issue by mgrabow1** [fguillot/picoDb#18]
It would be nice to be able to add columns to query, not just define the whole set in one place (useful eg. in kanboard plugins).
 A getter for the columns array (Table class) would be enough :)

**Comment by fguillot** [comment]
What do you mean? you can choose the desired columns: `$db->table('foobar')->columns('columnA', 'columnB')->findAll()`, is it what you are looking for?

**Comment by mgrabow1** [comment]
Not really. I'm thinking about a situation, where you already have a prepared PicoDB Table object, but want to add several columns to the query without redefining all of them.
A real example is a Kanboard plugin that expands the model eg. with reference hooks. I just needed to add 2 columns to a query (pagination:dashboard:task:query), but - because of this limitation - I had to redefine the whole column list which makes my plugin more prone to model changes.
